src/NLog.Layouts.GelfLayout/GelfConverter.cs

```diff
@@ -24,10 +24,10 @@ public class GelfConverter
 
         private string _hostName;
 
-        public void ConvertToGelfMessage(JsonWriter jsonWriter, LogEventInfo logEventInfo, IGelfConverterOptions converterOptions)
+        public void ConvertToGelfMessage(JsonWriter jsonWriter, LogEventInfo logEventInfo, IGelfConverterOptions converterOptions, Layout messageLayout)

```

Comment:
Use the `IGelfConverterOptions converterOptions` instead of adding a new parameter.